src/main/java/ch/njol/skript/expressions/ExprExperience.java

```diff
@@ -49,70 +50,77 @@
+		"\t\tadd 100 to dropped experience",
+		"on breed:",
+		"\tevent-father is a cow",
+		"\tset dropped experience to 10"
+})
+@Since("2.1, 2.5.3 (block break event), 2.7 (experience change event), INSERT VERSION (breeding event)")
+@Events({"experience spawn", "break / mine", "experience change", "entity breeding"})
 public class ExprExperience extends SimpleExpression<Experience> {
+
 	static {
 		Skript.registerExpression(ExprExperience.class, Experience.class, ExpressionType.SIMPLE, "[the] (spawned|dropped|) [e]xp[erience] [orb[s]]");
 	}
-	
+
 	@Override
-	public boolean init(final Expression<?>[] exprs, final int matchedPattern, final Kleenean isDelayed, final ParseResult parseResult) {
-		if (!getParser().isCurrentEvent(ExperienceSpawnEvent.class, BlockBreakEvent.class, PlayerExpChangeEvent.class)) {
-			Skript.error("The experience expression can only be used in experience spawn, block break and player experience change events");
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
+		if (!getParser().isCurrentEvent(ExperienceSpawnEvent.class, BlockBreakEvent.class, PlayerExpChangeEvent.class, EntityBreedEvent.class)) {
+			Skript.error("The experience expression can only be used in experience spawn, block break, player experience change and entity breeding events");
 			return false;
 		}
 		return true;
 	}
 	
 	@Override
 	@Nullable
-	protected Experience[] get(final Event e) {
-		if (e instanceof ExperienceSpawnEvent)
-			return new Experience[] {new Experience(((ExperienceSpawnEvent) e).getSpawnedXP())};
-		else if (e instanceof BlockBreakEvent)
-			return new Experience[] {new Experience(((BlockBreakEvent) e).getExpToDrop())};
-		else if (e instanceof PlayerExpChangeEvent)
-			return new Experience[] {new Experience(((PlayerExpChangeEvent) e).getAmount())};
+	protected Experience[] get(Event event) {
+		if (event instanceof ExperienceSpawnEvent)
+			return new Experience[] {new Experience(((ExperienceSpawnEvent) event).getSpawnedXP())};
+		else if (event instanceof BlockBreakEvent)
+			return new Experience[] {new Experience(((BlockBreakEvent) event).getExpToDrop())};
+		else if (event instanceof PlayerExpChangeEvent)
+			return new Experience[] {new Experience(((PlayerExpChangeEvent) event).getAmount())};
+		else if (event instanceof EntityBreedEvent)
+			return new Experience[] {new Experience(((EntityBreedEvent) event).getExperience())};
 		else
 			return new Experience[0];
 	}
 	
 	@Override
 	@Nullable
-	public Class<?>[] acceptChange(final ChangeMode mode) {
+	public Class<?>[] acceptChange(ChangeMode mode) {
 		switch (mode) {
+			case SET:
 			case ADD:
-			case DELETE:
 			case REMOVE:
-			case REMOVE_ALL:
-				return new Class[] {Experience[].class, Number[].class};
-			case SET:
-				return new Class[] {Experience.class, Number.class};
+			case DELETE:
 			case RESET:
-				return null;
+				return new Class[] {Experience[].class, Integer[].class};
```

Comment:
Why is this change?

Reply:
Actually unsure which one but this reminded me to fix `case SET` that shouldn't support multiple.
The rest of the change was mostly for cleanliness `remove_all` makes no sense for this expression it's single
`reset` makes more sense than `delete` or `clear`, same functionalities were kept

Reply:
My concerns was why changing from Number.class to Integer.class?
As for the multiple support and other cases, I think this will be a breaking change and should be in another PR

Reply:
Number to Integer shouldn't cause any issues but I'll double check to ensure, we have converters and they should handle it.
Besides that there shouldn't be any breaking changes, multiple and single support are the exact same besides `remove_all` being removed

Reply:
`REMOVE_ALL` being removed is considered a breaking change 😬 even tho it's barely used but we must notify users in changelog.

I guess if it's just that we can keep it in this PR and just state in the description why it's marked as breaking change.

As for the `Number -> Integer` nevermind about it, I double checked the javadocs and all methods accepts an Integer type except one accepting a Float but that's for CookingRecipe which we don't yet support

Reply:
Did want to double check the converters were working, yes even if I do `set dropped experience to 4.4` skript will convert it to `4`, so yeah only remove all is a breaking change